olp-cpp-sdk-core/include/olp/core/logging/Log.h

```diff
@@ -33,6 +33,21 @@
 #include <sstream>
 #include <string>
 
+/**
+ * @brief Temporary macros to ease EDGE_SDK_ prefix transition review.
+ */
+#define LOG_FUNCTION_SIGNATURE EDGE_SDK_LOG_FUNCTION_SIGNATURE
```

Comment:
Shouldn't this be at the end of file after EDGE_SDK_* are defined?

Reply:
Does it matter? These lines to removed soon any way and at the beginning of the file it catches an eye better.